sdk/attestation/azure-security-attestation/azure/security/attestation/_client.py

```diff
@@ -93,13 +92,13 @@ def get_signing_certificates(self, **kwargs):
 
     @distributed_trace
     def attest_sgx_enclave(self, quote, inittime_data=None, runtime_data=None, draft_policy=None, **kwargs):
-        # type:(bytes, Optional[AttestationData], Optional[AttestationData], Optional[str], Any) -> AttestationResponse[AttestationResult]
+        # type:(bytes, Optional[AttestationData], Optional[AttestationData], Optional[str], Dict[str, Any]) -> AttestationResponse[AttestationResult]
         """ Attests the validity of an SGX quote.
 
         :param bytes quote: An SGX quote generated from an Intel(tm) SGX enclave
         :param Optional[AttestationData] inittime_data: Data presented at the time that the SGX enclave was initialized.
         :param Optional[AttestationData] runtime_data: Data presented at the time that the SGX quote was created.
-        :param Optional[str] draft_policy: "draft", or "experimental" policy to be used with
+        :param Optional[str] draft_policy: "draft" or "experimental" policy to be used with
```

Comment:
```
:keyword draft_policy: ...
:paramtype str:
```

Reply:
Does it make sense to change inittime_data and runttime_data to be keyword args as well? I can see arguments either way.

Reply:
You can leave them in the signature, but if you add a default they should be documented as `keyword`s

Reply:
Ok, that makes sense. But doesn't it make the whole construct of "keywords" as somewhat loose? Instead of a "keyword" being "named parameters not explicitly listed in the parameters", it interprets keyword as "all named parameters".

Or does this only apply if the default is None?

Reply:
I believe the reason we do this (include default args in a signature) is it plays a little nicer with intellisense. This format will show up in the signature of the method in intellisense whereas if it's strictly a kwarg then it will show up in the documentation of intellisense. I realize that's confusing, does that make sense?

Reply:
Actually that makes perfect sense.